contracts/script/releases/v2.4.1-access-control-migration/upgrade.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "access-control-migration-v2.4.1",
```

Comment:
can we add an "info" field that would explain why this is needed and what it does?